packages/fx-core/resource/package.nls.json

```diff
@@ -935,5 +935,15 @@
   "driver.aadApp.error.missingEnv": "%s Environment variable %s is not set.",
   "driver.aadApp.error.generateSecretFailed": "Cannot generate client secret.",
   "driver.aadApp.error.unhandledError": "Unhandled error happened in %s action: %s",
-  "driver.aadApp.error.invalidFieldInManifest": "Field %s is missing or invalid in AAD app manifest."
+  "driver.aadApp.error.invalidFieldInManifest": "Field %s is missing or invalid in AAD app manifest.",
+  "driver.aadApp.log.startExecuteDriver": "Executing driver %s",
+  "driver.addApp.log.successExecuteDriver": "Driver %s executed successfully",
+  "driver.aadApp.log.failExecuteDriver": "Failed to execute driver %s. Error message: %s",
+  "driver.aadApp.log.startCreateAadApp": "Environment variable % not exist. Start creating a new AAD app.",
+  "driver.aadApp.log.successCreateAadApp": "Successfully created AAD app.",
+  "driver.aadApp.log.skipCreateAadApp": "Environment variable %s already exist. Skip creating new AAD app.",
+  "driver.aadApp.log.startGenerateClientSecret": "Environment variable % not exist. Start generating client secret for AAD app.",
+  "driver.aadApp.log.successGenerateClientSecret": "Successfully generated client secret.",
+  "driver.aadApp.log.skipGenerateClientSecret": "Environment variable %s already exist. Skip creating new AAD app.",
```

Comment:
should this be for generate client secret? 

Environment variable % does not exist, skipping AAD app client secret generation step.